tools/devtool

```diff
@@ -1507,10 +1518,45 @@ cmd_release() {
             post_release "$version" "$owner" "$repo" "$token" "$prerelease" "$release_text" ||
             delete_remote_tag "$version" "$remote" "Could not post release to GitHub."
             say "Draft release created successful. Go to https://github.com/$owner/$repo/releases to check it out!"
+
+            say "Updating \`latest\` release version tag..."
+            update_latest_tag "$version" "$remote" || die "Could not update latest tag."
     fi
 fi
 }
 
+# `Latest` tag is meant to be used for github latest tagging purposes.
+# This tag is always pointing to the largest version released so far.
+update_latest_tag() {
+    version="$1"
+    remote="$2"
+
+    # List all tags, ignoring the ones that contain `wip` string and fetch
+    # the one with the greatest version.
+    greatest_release=$(git tag | grep -E '^v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+' | sort -V | tail -1)
+
+    # Update latest tag only if the current release is the greatest version.
+    if [[ "$greatest_release" == "v$version" ]]; then
+        say "Creating latest tag against release $greatest_release..."
+
+        # Remove `latest` tag from local and also remote.
+        git tag -d latest
+        git push --delete "$remote" latest || die "Could not delete remote latest tag."
+
+        # Tag greatest release version with the `latest` tag and set the date
+        # to the date of the latest commit.
+        tag_msg="Latest Firecracker release: v$version."
+        GIT_COMMITTER_DATE="$(git show --format=%aD  | head -1)" git tag -a latest "v$version" -m "$tag_msg"
```

Comment:
Do not really understand this line. Where is GIT_COMMITTER_DATE used?
What command updates the date?

Reply:
`GIT_COMMITTER_DATE` environment variable is used by git internals to set dates to commits and tags. The line you highlighted temporarily sets the value for `GIT_COMMITTER_DATE`  to the date of the commit that HEAD currently points to AND creates the tag using that date.

The catch here is that by setting the variable and creating the tag on the same line (no `;` or `&&` used), `GIT_COMMITTER_DATE` is only temporarily overwritten and is visible only in the environment of the command that comes on the same line (that is the command that creates the tag). We do this because we do not want the date change to persist any further than while creating the latest tag.

Reply:
I understand it now. Cool!

What do you say we try to run this piece of code for updating the current latest tag for Firecracker which right now is wrongly pointing to 0.24.6 instead of 0.25.0.
As far as my understanding we only need to do this:
* rename the function from `update_latest_tag ` to `cmd_update_latest_tag `and then run `cmd_update_latest_tag v0.25.0 https://github.com/firecracker-microvm/firecracker.git`. is this so?